source/localizable/tutorial/deploying.md

```diff
@@ -0,0 +1,41 @@
+To deploy an Ember application simply transfer the output from `ember build` to a web server. This can be done with standard Unix file transfer tools such as `rsync` or `scp`. There are also services that will let you deploy easily.
```

Comment:
Hi @boztek, can you split up the sentences as specified in https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md#style-guide ? It makes it easier to review and diff. Appreciated :)